src/java.base/share/classes/com/sun/crypto/provider/CounterMode.java

```diff
@@ -53,6 +53,10 @@ class CounterMode extends FeedbackCipher {
     // number of bytes in encryptedCounter already used up
     private int used;
 
+    // chunkSize is a multiple of block size and used to divide up
+    // input data to trigger the intrinsic.
```

Comment:
CRC32 message digest computation algorithm is intrinsified at C1 and interpreter level to overcome this warmup related degradation.

Reply:
This comment looks incorrect, a method marked as an intrinsic is always inline expanded by C2 compile during parsing or during incremental inlining if -XX:+InlineIncrement is used.

I guess what you intend here is triggering an OSR compilation of loop by C2 compiler which in turn trigger intrinsic since C1 never intrinsifies crypto APIs